classes/module/Module.php

```diff
@@ -3516,6 +3541,23 @@ public function getMultistoreCompatibility(): int
     {
         return $this->multistoreCompatibility;
     }
+
+    /**
+     * In order to load or update the module's translations, we just need to clear SfCache.
+     * The translator service will be loaded again with the catalogue within the module
+     *
+     * @throws ContainerNotFoundException
+     */
+    private function loadBuiltInTranslations(): void
+    {
+        $modulePath = $this->getLocalPath();
+        $translationDir = sprintf('%s/translations/', $modulePath);
+        if (!is_dir($translationDir)) {
+            return;
```

Comment:
It should throw an exception, I see no reason to have a module without modules files :thinking:

Reply:
it's translations directory. Not all the modules have it